ansible/roles/nginx/templates/nginx.conf.j2

```diff
@@ -37,6 +37,7 @@ http {
         }
         location /api/v1 {
             proxy_pass http://{{ groups['controllers']|first }}:{{ controller.port }};
+            proxy_read_timeout 300s;
```

Comment:
Do we need little bit of slack, as the action, even when called blocking will return in 5 minutes + the time needed to propagate stuff?